spec/surfnperf_spec.js

```diff
@@ -1096,5 +1096,136 @@ define('spec/surfnperf_spec', [
 
     });
 
+
+    describe('#chromeLoadTimes', function() {
+      it('returns a reference to window.chrome.loadTimes() if the browser supports it, otherwise undefined', function() {
+        if(window.chrome && window.chrome.loadTimes) {
+          expect(SurfNPerf.chromeLoadTimes()).toEqual(window.chrome.loadTimes());
+        } else {
+          expect(SurfNPerf.chromeLoadTimes()).toEqual(undefined);
+        }
+      });
+    });
+
+    describe('#msFirstPaint', function() {
+      it('returns a reference to window.performance.timing.msFirstPaint if the browser supports it, otherwise undefined', function() {
+        if(window.performance && window.performance.timing && window.performance.timing.msFirstPaint) {
+          expect(SurfNPerf.msFirstPaint()).toEqual(window.performance.timing.msFirstPaint);
+        } else {
+          expect(SurfNPerf.msFirstPaint()).toEqual(undefined);
+        }
+      });
+    });
+
+    describe('#getFirstPaint', function() {
```

Comment:
add this between `describe('#getFirstPaint', function() {` & `describe('when the client is using Chrome', function() {` to ensure all these tests use the same values for the calculations:
```js
      beforeEach(function() {
        SurfNPerf._navigationTiming = true;
        spyOn(SurfNPerf, 'performanceTiming').andReturn({
          navigationStart: 1388595600000
        });
      });
```